test/test_helper.rb

```diff
@@ -38,3 +38,33 @@ def psych_load(*args)
 if defined?(GC.auto_compact) == "method"
   GC.auto_compact = true
 end
+
+# Basic test class where transformations will translate
+# based on difference between "value" attribute
+class TestAffineCoordinateSystem < RGeo::CoordSys::CS::CoordinateSystem
+  def initialize(value, dimension, *optional)
+    super(value, dimension, *optional)
+    @value = value
+  end
+  attr_accessor :value
+
+  def transform_coords(target_cs, x, y, z = nil)
+    ct = TestAffineCoordinateTransform.create(self, target_cs)
+    ct.transform_coords(x, y, z)
+  end
+
+  class << self
+    def create(value, dimension = 2)
+      new(value, dimension)
+    end
+  end
+end
+
+class TestAffineCoordinateTransform < RGeo::CoordSys::CS::CoordinateTransform
+  def transform_coords(x, y, z = nil)
+    diff = target_cs.value - source_cs.value
+    coords = [x + diff, y + diff]
+    coords << (z + diff) if z
+    coords
+  end
+end
```

Comment:
I think since this is used only in one file, I'd be more in favor of putting it directly in that file.

Or, we could have a `test/support` folder for those kinds of helpers, where we could also put the `data.wk{b,t}` that I made in my last PR

Reply:
Agreed. Moved it into the types_test file.